changelog.d/11988.bugfix

```diff
@@ -0,0 +1 @@
+Fix M_WRONG_ROOM_KEYS_VERSION errors not including the specced current_version field.
```

Comment:
```suggestion
Fix a long-standing bug where `M_WRONG_ROOM_KEYS_VERSION` errors would not include the specced `current_version` field.
```
We're trying to note when bugs were introduced, or mark them as "long-standing" in changelogs.